csharp/src/Drivers/Apache/Spark/SparkStatement.cs

```diff
@@ -50,6 +50,7 @@ protected override void SetStatementProperties(TExecuteStatementReq statement)
                 ComplexTypesAsArrow = false,
                 IntervalTypesAsArrow = false,
             };
+            statement.TrimArrowBatchesToLimit = true;
```

Comment:
is this only a Databricks feature or a general Spark feature?

Reply:
And if it's not available in Spark (or in all relatively-recent versions of Spark), then what's the impact of setting this? Would it cause an error or be silently ignored?

Reply:
Yes, this is Databricks only.

Reply:
> Is there some place you are using to check?

It's possible that we haven't yet tested this against a (non-Databricks) Spark instance :(. We'll follow up. Finding any kind of definitive knowledge about this API (and its multiple implementations) is pretty challenging.